entrypoint

```diff
@@ -21,7 +21,7 @@ EOF
 elif [ "z$1" = "zsdkbase" ] ; then
     echo "Pulling and tagging the base image"
     docker pull kbase/sdkbase
```

Comment:
Shouldn't this pull kbase/kbase:sdkbase2.latest?

Reply:
I don't really know the distinction between kbase/kbase.sdkbase.latest and kbase/sdkbase

Are they the same image, but under different tags?